### PR TITLE
Remove another engine hardcode from testcase upload.

### PR DESCRIPTION
**Pull request by oliverchang**

### CODE REVIEW

**Comment by oliverchang** [comment]
TBR

File: src/appengine/handlers/upload_testcase.py [hunk 1]
```diff
@@ -322,12 +322,10 @@ def do_post(self):
...
     fuzzer_name = request.get('fuzzer')
     job_type_lowercase = job_type.lower()
-    if 'libfuzzer' in job_type_lowercase:
-      fuzzer_name = 'libFuzzer'
-    elif 'afl' in job_type_lowercase:
-      fuzzer_name = 'afl'
-    elif 'honggfuzz' in job_type_lowercase:
-      fuzzer_name = 'honggfuzz'
+
+    for engine in fuzzing.ENGINES:
```
Comment (jonathanmetzman) [inline src/appengine/handlers/upload_testcase.py:326]: Kind of nervous someone might one day have multiple engine names in the job name.

**Review by jonathanmetzman — COMMENTED** [review]
LGTM

**Pull request merged**